x-pack/plugins/endpoint/package.json

```diff
@@ -6,9 +6,11 @@
   "license": "Elastic-License",
   "scripts": {},
   "dependencies": {
-    "react-redux": "^7.1.0"
+    "react-redux": "^7.1.0",
+    "seedrandom": "^3.0.5"
```

Comment:
This library seems to already be included in Kibana. https://github.com/elastic/kibana/blob/master/package.json#L255